### DISCUSSION TITLE
Chrono

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#4203] [General]
Hi simmers,

anyone else with chrono problems?

![grafik](https://user-images.githubusercontent.com/76531893/112995377-71b2bf00-916b-11eb-8add-0f5055a0e55c.png)

Greets klaus

**Comment by tracernz** [comment]
What is the problem you're having? The only thing I can see is you haven't set the ET timer to run.

**Reply by CptBayer** [reply]
Hi,

I started the chrono on the ND and the other one at the same time. i was wondering about hte time difference ?!

Greets Klaus

**Reply by tracernz** [reply]
Oh, yes, I see. Did you pause the sim or change the time of day, anything like that? Maybe one of them counts pauses or time jumps and the other doesn’t...

**Reply by CptBayer** [reply]
Hi,


no, no pause at all. But when you did a pause, the clock is running and running and is not going into a pause .... hmmmm... I recognized that by last flying a longer leg... Was wondering about hte flight time, instead of 4 hours it was round about 5 hours. It was inclusive the pause ... 

Greets Klaus

**Reply by tracernz** [reply]
> 
> 
> Hi,
> 
> no, no pause at all. But when you did a pause, the clock is running and running and is not going into a pause .... hmmmm... I recognized that by last flying a longer leg... Was wondering about hte flight time, instead of 4 hours it was round about 5 hours. It was inclusive the pause ...
> 
> Greets Klaus

If you can give a nice easy way to reproduce the issue, then [open a GitHub issue](https://github.com/flybywiresim/a32nx/issues/new?assignees=&labels=Bug&template=bug_report.md) with your instructions to reproduce e.g.
- Spawn at EDDF runway XX
- Set F-PLAN to blah blah blah, X SID
- Start both chronos and set ET timer to run
- Takeoff and begin climb
- Pause sim for 2 minutes
- Check times are different

This way it won't get lost and we can fix it at a later date.

**Comment by Annolago** [comment]
You where probably using active pause. The sim will continue to run but your plane will freeze in place - hence then clock will continue to run. you can't do it differently if you are online with other people flying. Otherwise planes would randomly appear out of no where just due to them taking a bad room brake.
